current_spring2026/ingestion/ingest.py

```
@@ -0,0 +1,411 @@
+        for record in iter_metadata_records(metadata_file, seen_record_ids):
+            if not is_valid(record):
+                skipped += 1
+                continue
+            batch.append(record)
+            if len(batch) >= batch_size:
+                flush(batch)
+                batch = []
+        if batch:
+            flush(batch)
+
+    print(f"\n✓ Ingestion complete.")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)